docs/Configuration.rst

```diff
@@ -5981,7 +6036,171 @@ data is stored in the database.
 To disable the login history data storage in the database, simply set the parameter 
 ``exo.audit.login.enabled`` to ``false``.
 
+.. _Configuration.Wallet:
+
+======================
+Configuring eXo Wallet
+======================
+
+The eXo Wallet addon uses the `Ethereum Blockchain <https://www.ethereum.org/>`__ to let eXo users hold and exchange crypto-currency (aka tokens) managed inside this blockchain.
+
+Blockchain network settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addon is configured by default to communicate with Ethereum's public network (called the "Mainnet") throught `Infura <https://infura.io/>`__ using websocket and http protocols:
+
+.. code-block:: jproperties
+
+    exo.wallet.blockchain.networkId=1
+    exo.wallet.blockchain.network.websocket=wss://mainnet.infura.io/ws/v3/a1ac85aea9ce4be88e9e87dad7c01d40
+    exo.wallet.blockchain.network.http=https://mainnet.infura.io/v3/a1ac85aea9ce4be88e9e87dad7c01d40
+
+.. note:: 
+
+    The default configuration points to an eXo-owner infura project ID that serves as proxy to the mainnet.
+    To learn more about infura infrastructure and projects, please visit `Infura documentation <https://infura.io/docs/>`__
+
+- The Websocket endpoint is used to:
+
+   -  Scan newly mined blocks on the blockchain to check if any validated transaction involves a wallet owned by a user of your eXo Platform instance.
+      The periodicity can be configured using the following property (Default value: every hour):
```

Comment:
of your eXo Platform instance => of your eXo Platform instance is available

Reply:
changed to:

`owned by an available user in your eXo Platform instance.`

Reply:
done